### ISSUE TITLE
No .APP file included in download - I don't see it in the Files tab either for the most recent update 

### ISSUE THREAD

**Issue by esmrlnd** [GeoPyTool/GeoPyTool#16]
### Bug report

**Bug summary**

 The .APP file was not downloaded per the picture examples onto my MAC. 

**GeoPyTool version**
 I just downloaded the code yesterday from the green code button
  * Operating system: OS Catalina 10.15.3
  * GeoPyTool version: 
  * Which function causes the error:
  * Other Information: 

  * 操作系统（精确到版本号）:
  * GeoPyTool 版本: 
  * 出错的功能:
  * 其他信息: 

  
 From the code button download on github

  
  
**Data for reproduction**

**Actual outcome**

```
# If applicable, paste the console output or the Screenshot of terminal during the error here 
#
#
```

**Comment by cycleuser** [comment]
Yes, sadly that I cannot afford a new macbook and have no way to build a macos app any more.

Such a sad story.
